src/library/Uno.Material/Controls/Chips/Chip.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls.Primitives;
+
+namespace Uno.Material.Controls
+{
+	public partial class Chip : ToggleButton
+	{
+		/// <summary>
+		/// Fires when a ToggleButton is checked or unchecked, except when set with <see cref="SetIsCheckedSilently"/>.
+		/// </summary>
+		internal event RoutedEventHandler IsCheckedChanged;
+
+		#region DependencyProperty: IsCheckable = true
```

Comment:
I understand the name but I think we can find better =D

Reply:
IsCheckEnabled ?

Reply:
That sounds wordy. Microsoft seems to agree with me:
https://www.google.com/search?q=ischeckable+site%3Adocs.microsoft.com
https://www.google.com/search?q=ischeckenabled+site%3Adocs.microsoft.com

Reply:
Sold :)